.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2.1
+
+orbs:
+  node: circleci/node@5.1.0
+
+jobs:
+  lint-and-test:
+    docker:
+      - image: "cimg/base:stable"
+    steps:
+      - checkout
+
+      # Install node version from .nvmrc
+      - node/install
+
+      - run: npm ci
+      - run: npm run format-check
+      - run: npm run lint
+      - run: npm run build
+      - run:
+          name: Build output is up-to-date
+          command: |
+            if ! [[ -z "$(git status --porcelain)" ]]; then 
```

Comment:
Just one question may be a noob question, what does this command check?

Reply:
If there are uncommited files in the repo, the CI job is exited with status code `1` (error). That way we ensure the `npm run build` step does not cause any changes and the `dist` folder is already up-to-date.

`git status --porcelain` is a more script-friendly version of `git status`